Simplenote/NoteEditorViewController.m

```diff
@@ -526,11 +526,18 @@ - (void)unpublishNote
 - (BOOL)validateMenuItem:(NSMenuItem *)menuItem
 {
     // Note menu
+    if (menuItem == deleteItem || menuItem == printItem) {
+        return [[SimplenoteAppDelegate sharedDelegate] isMainWindowVisible];
+    }
     if (menuItem == newItem) {
         return !self.viewingTrash;
-    } else if (menuItem == deleteItem) {
+    }
+
+    if (menuItem == deleteItem) {
```

Comment:
This code is never reached now, right? The options are now enabled when I'm viewing the trash.